homeassistant/components/signal_messenger/manifest.json

```
@@ -3,6 +3,6 @@
   "name": "Signal Messenger",
   "documentation": "https://www.home-assistant.io/integrations/signal_messenger",
   "codeowners": ["@bbernhard"],
-  "requirements": ["pysignalclirestapi==0.3.4"],
+  "requirements": ["pysignalclirestapi==0.3.17"],
```

```suggestion
  "requirements": ["pysignalclirestapi==0.3.17", "future==0.18.2"],
```
Seems that the `future` package is also required due to the Python 2.7 backport with this particular package. The lib  itself does not define its own requirements.

---

Ah I was wondering why I had to install that library separately! I had similar issues with `sqlalchemy` and `zeroconf` - do you know if those need to be added here too? Or were those missing packages something unrelated?

---

If that is with HA, I haven't encountered those issues. If you are talking generally, most packages define `REQUIRES` so pip can automatically install dependencies. This particular package does which causes this issue.

---

It seems that this issue has been resolved by the library with the newest version. Suggest bumping the library and removing the future dependency. https://github.com/bbernhard/pysignalclirestapi/releases/tag/0.3.18